QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KubeStellar**](#1-install-kubestellar-pre-requisites)
```

Comment:
kcp is NOT part of kubestellar.  The title of step 1 should not mislead the reader about this.  I think something like the following would be better.

> Install and run **kcp** and **KubeStellar**

Reply:
See the discussion here: https://github.com/kcp-dev/edge-mc/pull/350#discussion_r1193864780 . Andy and I agree that we should not include kcp in that title section.

Reply:
We agreed to leave KCP out of the title

Reply:
I disagree.  While the focus of this document is indeed KubeStellar, omitting mention of kcp from the title of this section tells the reader that kcp is part of KubeStellar --- and that is the wrong message.

Reply:
@clubanderson any comment on this? just want to make sure that we have an agreement before making any change.